nasa-gibs/worldview
# WV-3042 Support TEMPO layer

## Description

Support TEMPO as a Granule Browse Layer

## How To Test

1. `git checkout WV-3042-TEMPO`
2. `npm run build`
3. `npm run watch`
4. Go to this [link](http://localhost:3000/?v=-282.20169389378555,-182.2890036386939,295.5827936913064,138.36801233011576&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,VIIRS_NOAA20_CorrectedReflectance_BandsM3-I3-M11_Granule(count=10),VIIRS_NOAA20_CorrectedReflectance_TrueColor_Granule(count=10),VIIRS_SNPP_CorrectedReflectance_BandsM11-I2-I1_Granule(count=10),VIIRS_SNPP_CorrectedReflectance_TrueColor_Granule(count=10),VIIRS_SNPP_CorrectedReflectance_BandsM3-I3-M11_Granule(count=10),VIIRS_NOAA20_CorrectedReflectance_BandsM11-I2-I1_Granule(hidden,count=10),VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor(hidden)&lg=true&t=2024-02-28-T13%3A11%3A41Z)
5. Ensure that the Granule browse layers work as expected.
